admission/hooks/mutate_httpproxy.go

```diff
@@ -0,0 +1,59 @@
+package hooks
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	contourv1 "github.com/projectcontour/contour/apis/projectcontour/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	annotationKubernetesIngressClass = "kubernetes.io/ingress.class"
+	annotationContourIngressClass    = "projectcontour.io/ingress.class"
+	annotationIngressClassDefault    = "forest"
```

Comment:
It would be better if this is not a constant, but a variable that
is given by a command-line flag because:

- `forest` is too specific.
- We may change our minds.